.teamcity/settings.kts

```diff
@@ -0,0 +1,79 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
+
+version = "2021.2"
+
+project {
+    vcsRoot(RhinoLicensingVcs)
+
+    buildType(RhinoLicensingBuild)
+}
+
+object RhinoLicensingBuild : BuildType({
+    name = "Rhino.Licensing Build"
+
+    artifactRules = "code_drop/TestResults/issues-report.html"
+
+    params {
+        param("teamcity.git.fetchAllHeads", "true")
+        param("env.Git_Branch", "${RhinoLicensingVcs.paramRefs.buildVcsBranch}")
+        param("env.vcsroot.branch", "%vcsroot.branch%")
+        text("env.NUGET_DEV_SOURCE", "https://hermes.chocolatey.org:8443/repository/choco-internal-testing/", readOnly = true, allowEmpty = true)
+        text("env.NUGET_PROD_SOURCE", "https://hermes.chocolatey.org:8443/repository/choco-internal-testing/", readOnly = true, allowEmpty = true)
```

Comment:
The root project has `env.NUGETDEV_SOURCE` (and `env.NUGETDEV_API_KEY`) parameters that will be available if you'd prefer to use that/those. If so, remove these lines and change references in the actual code (i.e. `NUGET_DEV_SOURCE` -> `NUGETDEV_SOURCE`)

This isn't a hard requirement, just making sure you're aware.